src/main/java/org/kaazing/github/Action.java

```diff
@@ -194,4 +211,168 @@ public String getDescription() {
         }
     };
 
+    public static Action LIST_ISSUES_CLOSED_WITHIN_DAY = new Action() {
+
+        @Override
```

Comment:
Minor: There are formatting issues with whitespace here, I can show you how we do code formatting at kaazing if you stop by sometime